samples/helloworld/kitex/client/main.go

```diff
@@ -8,16 +8,27 @@ import (
 	dubbo "github.com/kitex-contrib/codec-dubbo/pkg"
 	"github.com/kitex-contrib/codec-dubbo/samples/helloworld/kitex/kitex_gen/hello"
 	"github.com/kitex-contrib/codec-dubbo/samples/helloworld/kitex/kitex_gen/hello/greetservice"
+	"github.com/kitex-contrib/obs-opentelemetry/provider"
+	"github.com/kitex-contrib/obs-opentelemetry/tracing"
 )
 
 func main() {
+	serviceName := "helloworld-client"
+	p := provider.NewOpenTelemetryProvider(
+		provider.WithServiceName(serviceName),
+		provider.WithExportEndpoint("localhost:4317"),
+		provider.WithInsecure(),
+	)
+	defer p.Shutdown(context.Background())
+
 	cli, err := greetservice.NewClient("helloworld",
-		client.WithHostPorts("127.0.0.1:21001"),
+		client.WithHostPorts("127.0.0.1:21000"),
```

Comment:
port 21001 is used to invoke dubbo-java side in samples/helloword/dubbo.
For testing, I would add this case in https://github.com/kitex-contrib/codec-dubbo-tests

Reply:
> port 21001 is used to invoke dubbo-java side in samples/helloword/dubbo. For testing, I would add this case in https://github.com/kitex-contrib/codec-dubbo-tests

get

Reply:
> port 21001 is used to invoke dubbo-java side in samples/helloword/dubbo. For testing, I would add this case in https://github.com/kitex-contrib/codec-dubbo-tests

done